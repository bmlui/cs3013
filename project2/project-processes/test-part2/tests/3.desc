Test external command with arguments
