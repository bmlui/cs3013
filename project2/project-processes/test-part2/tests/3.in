find . -name 1.in
exit

