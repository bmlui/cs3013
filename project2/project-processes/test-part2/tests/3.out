test-part2$ find . -name 1.in
./tests/1.in
test-part2$ exit
